packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -817,13 +815,69 @@ class StackViewport extends Viewport implements IStackViewport {
     this.setVOIGPU(voiRange, suppressEvents);
   }
 
-  private setRotation(rotationCache: number, rotation: number): void {
+  getRotation = (): number => {
+    if (this.useCPURendering) {
+      return this.getRotationCPU();
+    } else {
+      return this.getRotationGPU();
+    }
+  };
+
+  private getRotationCPU = (): number => {
```

Comment:
I suspect CPU rendering has issues with flip and rotate as well which I did not address. Should I?